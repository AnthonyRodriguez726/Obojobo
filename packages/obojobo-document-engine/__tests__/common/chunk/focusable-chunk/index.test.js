import React from 'react'
import FocusableChunk from '../../../../src/scripts/common/chunk/focusable-chunk'
import renderer from 'react-test-renderer'

describe('FocusableChunk', () => {
	test('Renders as expected', () => {
		const component = renderer.create(<FocusableChunk />)
		let tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})

	test('with indent and spellcheck', () => {
		const component = renderer.create(
			<FocusableChunk indent={2} spellcheck={false} className="test-123" />
		)
		let tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
