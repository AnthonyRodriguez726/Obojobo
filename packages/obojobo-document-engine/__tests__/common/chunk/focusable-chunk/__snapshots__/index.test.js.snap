// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FocusableChunk 1`] = `
<div
  className="focusable-chunk anchor-container"
  contentEditable={false}
>
  <span
    className="anchor"
    contentEditable={true}
    data-group-index="anchor:main"
    indent={0}
    name="main"
    spellcheck={true}
    suppressContentEditableWarning={true}
    tabIndex=""
  />
</div>
`;

exports[`FocusableChunk with indent and spellcheck 1`] = `
<div
  className="focusable-chunk anchor-container test-123"
  contentEditable={false}
>
  <span
    className="anchor"
    contentEditable={true}
    data-group-index="anchor:main"
    indent={2}
    name="main"
    spellcheck={false}
    suppressContentEditableWarning={true}
    tabIndex=""
  />
</div>
`;
